.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
```

Comment:
This is the short list for edgex-docs. Please update with full checklist.

Reply:
@rsdmike could you please update this PR according to Lenny' request?  Thanks.

Reply:
@rsdmike could you please help resolve this PR?